inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -16,20 +16,20 @@ params:
   r_estim_window:
     label: "Number of days to include to get the latest observed value of Rt."
     value: 21
-  use_epiparameter: 
+  use_epiparameter_database: 
     label: "Should the serial interval distribution be extracted directly from the epiparameter package?"
     value: FALSE
-  epiparameter_pathogen: 
-    label: "Name of the pathogen in the epiparameter database if `use_parameter = TRUE`."
-    value: "SARS_CoV_2_wildtype"
+  disease_name: 
```

Comment:
This is not so much a comment on these changes, but if it would be useful to be able to read `<epidist>`s by specifying pathogen name let me know and I can add this to {epiparameter}.

Reply:
I think it's probably useful for users as it's likely that they'll want to focus on a specific pathogen, especially for those diseases where each pathogen has a distinct clinical presentation or characteristics... otherwise is there a way they could specify the pathogen when creating the epidist object? or not really?

Reply:
If they create the `<epidist>` themselves with `epidist()` they can specify the pathogen. But cannot select by pathogen using `epidist_db()`. Please can you make an issue in the {epiparameter} repository describing this feature and requesting it be added to the package? I'm happy to work on adding it in the near future.

Reply:
Added to issue https://github.com/epiverse-trace/epiparameter/issues/246

Reply:
Thanks!